README.md

```diff
@@ -25,6 +25,18 @@ Make sure you have GCC-4.6, G++-4.6, and GFortran-4.6 installed. For
 a full list of external dependencies on Ubuntu you can look at our
 [Travis configuration](https://github.com/graalvm/sulong/blob/master/.travis.yml).
 
+On the Mac you can use Homebrew:
+
+    brew install gcc
```

Comment:
Under Ubuntu I have to install a separate package `gfortran-4.6`. Maybe the Homebrew `gcc` package just groups several GCC projects together?